test/modules/carto/config.spec.js

```diff
@@ -1,35 +1,36 @@
 import test from 'tape-catch';
 import {getDefaultCredentials, setDefaultCredentials} from '@deck.gl/carto';
 
-test('auth#getDefaultCredentials', t => {
+test('config#getDefaultCredentials', t => {
   const credentials = getDefaultCredentials();
 
   t.notOk(credentials === null, 'default credentials available');
   t.end();
 });
 
-test('auth#getDefaultCredentials', t => {
+test('config#getDefaultCredentials', t => {
   // partial (keeping other params)
   setDefaultCredentials({username: 'a-new-username'});
   let credentials = getDefaultCredentials();
   t.ok(credentials.username === 'a-new-username', 'user update');
   t.ok(credentials.apiKey === 'default_public', 'keep default apiKey');
-  t.ok(
-    credentials.serverUrlTemplate === 'https://{user}.carto.com',
-    'keep default serverUrlTemplate'
-  );
+  t.ok(credentials.mapsUrl === 'https://{user}.carto.com/api/v1/map', 'keep default mapsUrl');
```

Comment:
How about adding also the 'region' to the specs?

Reply:
As we're just inserting and fetching an object it just make the test bigger without adding too much value. 

In fact, we should only keep 1, otherwise we're just testing object aassignation in JS 😄

Reply:
Just in case we change this in the future, but yes... it doesn't matter much